exercises/practice/paasio/.meta/src/reference/java/Paasio.java

```diff
@@ -0,0 +1,140 @@
+import java.io.*;
```

Comment:
Please don't use `*` imports. For example:

```suggestion
import java.io.Closeable;
import java.io.InputStream;
import java.io.OutputStream;
```